server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -157,10 +151,23 @@ private void logFailedCheck(Result result) {
         }
     }
 
-    private List<Result> retrieveUnhealthyOrDegradedHealthChecks() {
+    private ImmutableList<Result> retrieveHealthChecks() {
```

Comment:
1. It is not just a `retrieve` but it perform some side effects

And

2. The return type is not health checks but results.

Hence I propose to name this `executeHealthChecks`